FogCreek/Glitch-Community
# Sweet line: collection projects cache

## Links
* Remix link: https://sweet-line.glitch.me

## Changes:
* collection items fetch their own projects, curators
* adding a project to a collection on the teams page will now update the collection
* create collection project cache, provider, hooks

## How To Test:
* sign into https://sweet-line.glitch.me
* visit your user page
* create a collection
* add a project to that collection
* the collection item should update to include that project
* repeat the steps above but with a team page
* verify More From Glitch loads on https://sweet-line.glitch.me/@glitch/glitch-this-week-june-19-2019
* verify Included in Collections loads on https://sweet-line.glitch.me/~love-and-seduction
* verify collection results load on https://sweet-line.glitch.me/search?q=react&activeFilter=collection
